src/Integration.Vsix/Analysis/AnalysisConfigMonitor.cs

```diff
@@ -38,24 +39,26 @@ internal interface IAnalysisConfigMonitor
     internal sealed class AnalysisConfigMonitor : IAnalysisConfigMonitor, IDisposable
     {
         private readonly IAnalysisRequester analysisRequester;
-        private readonly IUserSettingsProvider userSettingsProvider;
         private readonly IActiveSolutionBoundTracker activeSolutionBoundTracker;
```

Comment:
I haven't removed IActiveSolutionBoundTracker because unfortunately it is still needed in OnUserSettingsChanged.

Reply:
At least it isn't listening to the events.
The `IConfigurationProvider` provides a method to get the current binding info (and is what the active solution bound tracker uses internally). Unfortunately, it's a "local service" so it can't be directly imported.

Reply:
For the avoidance of doubt, I'm not suggesting switching to use the `IConfigurationProvider` now.

Reply:
Yea, I took a look at it, it won't be simple to MEF-it.